resources/dicts/thoughts/alive/kitten.json

```
@@ -420,7 +589,9 @@
             "Is in r_c's way... again",
             "Yowls loudly in the middle of the clearing for absolutely no reason",
             "Begs for life-saving herbs to cure {PRONOUN/m_c/poss} minor scrape",
-            "Is calling a Clan meeting, before r_c rudely interrupts {PRONOUN/m_c/object}"
+            "Is calling a Clan meeting, before r_c rudely interrupts {PRONOUN/m_c/object}",
+            "Is demanding r_c's attention even though {PRONOUN/r_c/subject} {VERB/r_c/are/is} busy",
```

```suggestion
            "Is demanding r_c's attention even though {PRONOUN/r_c/subject}{VERB/r_c/'re/'s} busy",
```
minor nitpick, flows slightly better this way (feel free to ignore this suggestion if you want)